### PR TITLE
Setup docusaurus website based on markdown documents

### PR DESCRIPTION
**Pull request by SamChou19815**
Click on the deploy-preview job to see the deployed site.

### CODE REVIEW

File: src/pages/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- [Basic Conditions](/docs/basic-conditions) specifies a list of conditions that Computerization
+  must agree and enforce consistently.
+- [Benefits and The Revision Process](/docs/benefits) specifies a list of benefits that
+  Computerization can receive. Some benefits can only be claimed by enforcing certain good
+  engineering practices.
+- [Approved Reimbursement List](/docs/approved-reimbursement-list) specifies a list of eligible
+  operating cost reimbursements for Computerization.
+- [`SIGNATURE.txt`](/SIGNATURE.txt) lists the GitHub IDs of people who signed the document. Signing
+  in that file expresses your approval of this agreement and it is a prerequisite to make
+  modifications to _  other files.
```
Comment (yechs) [inline src/pages/index.md:26]: There's an extra "_" here that should be removed

**Review by yechs — COMMENTED** [review]
A little bug here

Also, I think it may make more sense to include the "introduction" page (`src/pages/index.md`) in the sidebar, but I'm not sure whether it's possible in docusaurus.

**Comment by SamChou19815** [comment]
> A little bug here
> 
> Also, I think it may make more sense to include the "introduction" page (`src/pages/index.md`) in the sidebar, but I'm not sure whether it's possible in docusaurus.

Done using docs only mode.

**Comment by yechs** [comment]
LGTM! Awaiting approvals from the current presidents

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**